CHANGELOG.md

```diff
@@ -3,6 +3,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.7.0] - 2024-11-19
+### Changed
+- Updated the hms namespaces for metrics for both readwrite and readonly.
+### Added
+- Added following variables:
+  - `hms_ecs_metrics_readwrite_namespace`
```

Comment:
Why you need separate vars for ecs or EKS can combine the two?

Reply:
ah for backwards compatibility I see.

Reply:
Yes for backward compatibility, with this change we are going to have the same namespace.